cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -83,6 +83,14 @@
   end
 end
 
+When(/^I authenticate as "([^"]*)" with account "([^"]*)"/) do |login, account|
+  user = lookup_user(login, account)
+  user.reload
+  steps %Q{
```

Comment:
This is one of things I'll be putting in my best practices docs, but `steps` is considered an anti-pattern.  From the cucumber docs:

> While it is possible to call steps from step definitions in Cucumber-Ruby,
> this is not recommended.  The best way to achieve composition and reuse, is to
> use the features of your programming language. If you want to combine several
> actions into one step, extract individual (helper) methods and call these
> methods from your step definition.

Reply:
That makes sense, and I will refactor this.